superset/views/core.py

```diff
@@ -673,12 +650,6 @@ def save_or_overwrite_slice(  # noqa: C901
                 dashboard_title=request.args.get("new_dashboard_name"),
                 owners=[g.user] if g.user else [],
             )
-            flash(
-                _(
-                    "Dashboard [{}] just got created and chart [{}] was added to it"
```

Comment:
tested that chart adding to dashboard is handled frontend now and flashed properly without these obsolete flash messages